### DISCUSSION TITLE
Easy Watermark on pictures from nemo or others

### DISCUSSION THREAD

**Discussion by jonathan-berthet** [linuxmint/discussions#511] [Ideas]
Hi,

It's always complicated to add a watermark to a picture. You can use Gimp or upload your file on the Internet (not very secure...). I think it shall be a good idea to have a one-click action to create a watermark on one or multiple pictures directly from the file explorer. We will need to choose the text, the size, the style, the orientation, ... It have to be easy to use, with a live-view of the result (a little windows which show in live what will be the final result).

I do not have development competency, but I can be a beta-tester if needed.

Maybe that it can be extended to videos?

Regards.

Vanmas

**Comment by Jopp-gh** [comment]
Should be doable with a Nemo script.
Here's an article how to build the code (assuming you use Mint and ImageMagick is installed too):
https://www.the-art-of-web.com/system/imagemagick-watermark/
